### PR TITLE
test(validate): imporve test coverage of the the validate package 1336

### PR DESCRIPTION
**Pull request by sdeli**

### CODE REVIEW

File: packages/validate/test/validate.spec.ts [hunk 1]
```diff
@@ -3,8 +3,40 @@ import { validate } from '../src/index';
 
 describe('ValidateSchema', function () {
   describe('EN10168 types', () => {
-    const validCertificatePath = `${__dirname}/../../../fixtures/EN10168/v0.0.2/valid_cert.json`;
-    const invalidCertificatePath = `${__dirname}/../../../fixtures/EN10168/v0.0.2/invalid_cert.json`;
+    const certVersion = 'v0.0.2';
```
Comment (sdeli) [inline packages/validate/test/validate.spec.ts:6]: Variables created to satisfy linters need for no duplicate string literals.

File: packages/utils/test/utils.spec.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  it('should validate ecoc certficates', async () => {
+    const ecocSchema = JSON.parse(readFileSync(ECOC_CERT_PATH, 'utf8') as string);
+
+    const responseOk = asECoCCertificate(ecocSchema, '/');
+    expect(responseOk).toHaveProperty('ok');
+
+    delete ecocSchema.RefSchemaUrl;
+    const responseInvalid = asECoCCertificate(ecocSchema, '/');
+    expect(responseInvalid).toHaveProperty('error');
+  }, 25000);
```
Comment (getlarge) [inline packages/utils/test/utils.spec.ts:17]: Why did you set a so long timeout ?

File: packages/validate/src/index.ts [hunk 3]
```diff
@@ -27,15 +28,20 @@ const validateOptions: ValidateOptions = {
...
     .filter((name: string) => !ignoredPaths.includes(name) && ignoredExts.every((ext) => !name.endsWith(ext)))
     .map((dir: string) => path.resolve(localSchemasDir, dir)) as string[];
 
   const subDirectories = await Promise.all(
-    directories.map(async (dir) => {
-      return (await readDir(dir))
+    dirsAndFiles.map(async (dirOrFile) => {
+      const isFile = fs.lstatSync(dirOrFile).isFile();
+      if (isFile) {
+        return dirOrFile;
```
Comment (getlarge) [inline packages/validate/src/index.ts:39]: Here the file is always an absolute path right ?
Otherwise you might try to resolve it.

**Review by getlarge — COMMENTED** [review]
That's some nice extras, i think the `utils` still requires some extra tests.
You should take care of your git config to commit with your github account and also add your public GPG key in Github settings.

File: packages/utils/test/utils.spec.ts (hunk 2 above)
Comment (sdeli) [reply packages/utils/test/utils.spec.ts:17]: thats a mistake

Comment (sdeli) [reply packages/utils/test/utils.spec.ts]: done

**Pull request closed without merging**